src/index.js

```diff
@@ -63,3 +64,7 @@ export default (options: options = {}): Promise<any> => {
   });
 
 };
+
+animateValue.easings = Object.assign({}, namedEasings);
+
```

Comment:
I want some default easing available, so bundling them for the dist isn't that much of a problem for me.
Maybe we can export them as a seperate module? (via Lerna?)